src/cluster/websocket/const.bif

```diff
@@ -0,0 +1,2 @@
+##! C++ access for consts from the websocket subsystem.
+const Cluster::websocket_dispatcher_queue_size: count;
```

Comment:
I do not know about the existing code, but is this a standard pattern of using bifcl to emit C++ forward declarations? I would have expected for us to just get the constant declared in script land in `init-bare.zeek` dynamically. The danger of decoupling from that is that we now need to maintain name and type in two places.

Reply:
Thanks - I'll move the queue size to `WebSocketServerOptions` instead. That allows to move the default into `cluster/main.zeek` and also makes it configurable on a per server.